style.css

```diff
@@ -0,0 +1 @@
+h1 { color: pink; }
```

Comment:
- [ ] Here you are using an element selector, but you are supposed to use a class selector. Please add a class in your heading and use that class to style your heading.